cc-mt-service/.env.example

```diff
@@ -1 +1,9 @@
+<<<<<<< HEAD
```

Comment:
this file has merge conflicts, it should be resolved. Only one port and remove the other merge conflict related stuff (<<<<< HEAD, =======, >>>>>>> 043e03....)